server/models/draftReferral.ts

```diff
@@ -23,6 +23,7 @@ export interface ReferralFields {
   whenUnavailable: string | null
   serviceUser: ServiceUser
   maximumEnforceableDays: number
+  reasonForChange?: string | null
```

Comment:
not sure about this - draft referrals don't have a reason for change so this feels like we are forcing this field on just to make it work